contracts/modules/cover/LimitOrders.sol

```diff
@@ -127,16 +130,14 @@ contract LimitOrders is ILimitOrders, MasterAwareV2, EIP712 {
     emit OrderCancelled(id);
   }
 
-  /// @notice Handles verification of the order signature
+  /// @notice Returns the hash of the structured data of the order
```

Comment:
`struct` is ambiguous, `build` is a bit misleading too, why not call the function `_getOrderHash`?

Reply:
I would also include the `_hashTypedDataV4` function call here as well so you'd return the actual digest directly and can call the function `getOrderId`. The execution details hashing may be moved here as well. The `_extractIdAndSigner` function can then be removed as the only extra thing it does is `ECDSA.recover` which can be inlined. I would also make the `getOrderId` function public as it may be useful for testing but could be used by other contracts.